Tests/SentryTests/Transaction/SentrySpanTests.swift

```diff
@@ -63,6 +63,131 @@ class SentrySpanTests: XCTestCase {
         clearTestState()
     }
     
+    func testSpanDoesNotIncludeTraceProfilerID() throws {
+        fixture.options.profilesSampleRate = 1
+        let span = fixture.getSut()
+        XCTAssert(SentryTraceProfiler.isCurrentlyProfiling())
+        span.finish()
+        
+        let serialized = span.serialize()
+        XCTAssertNil(serialized["profile_ids"])
+    }
+    
+    // test that a span that starts before a continuous profile and ends before
+    // the profile ends includes that profile id in the span
+    func testSpanWithProfileId() throws {
+        let span = fixture.getSut()
+        SentryContinuousProfiler.start()
+        let profileId = try XCTUnwrap(SentryContinuousProfiler.profiler()?.profilerId.sentryIdString)
+        span.finish()
+        
+        let serialized = span.serialize()
+        
+        let includedProfileIDs = try XCTUnwrap(serialized["profile_ids"] as? [String])
+        XCTAssert(includedProfileIDs.contains(profileId))
+    }
+
+    // test that a span that starts before a continuous profile and ends after
+    // the profile ends includes that profile id in the span
+    func testSpanWithProfileId1() throws {
```

Comment:
`l`: The method name could better reflect what it's testing, so when a test fails, you know what's wrong without reading the comment above.

Reply:
yep i updated them some time after you left this review, check em out now, they also have ascii diagrams!